examples/nextjs/pages/index.tsx

```diff
@@ -1,3 +1,4 @@
+import React from 'react';
```

Comment:
__*Comment*__

Worth noting we could consider just disabling this rule. Modern React doesn't require you to import it to each file unless you are specifically using something from it. Though we are supporting older versions so I could go either way on this.

Reply:
I've been trying to disable this rule, but I'm not getting lucky. I suspect we're in a very old eslint react plugin version because the error is being reported as:
```
'React' is not defined  react/jsx-no-undef
```
Where in modern lint it should be:
```
'React' must be in scope when using JSX react/react-in-jsx-scope
```

I'm unable to update eslint at this point because the package.json deps are broken. I'm gonna fix this in the next PR.